node/tests/hubController-testConfig.json

```diff
@@ -17,11 +17,11 @@
         "resource": "power",
         "resourceBlob": {"value":true}
     },
+    "subscriptionInfo": {
+        "callbackUrl": "http://drsubscriptiontest.azurewebsites.net/subs",
+        "key": "imalwaysangry"
+    },
     "subscription": {
```

Comment:
[https://github.com/openT2T/cloud/issues/17](https://github.com/openT2T/cloud/issues/17) ==> FYI assigned this issue to you -- can you confirm all 10 tests are passingwith your fix now? thanks!